strategies/changed/src/main/java/org/arquillian/smart/testing/vcs/git/GitChangesDetector.java

```diff
@@ -98,6 +98,10 @@ boolean matchPatterns(String path) {
         return false;
     }
 
+    boolean matchMainClassesPattern(String path) {
+        return matchPattern(path, "**/src/main/java/**/*.java");
```

Comment:
then I think it would be good to explicitly add it in `<includes>` of surefire configuration, Do we have any other way to handle it for other tools as well?